### PR TITLE
Add TPS banner

### PR DESCRIPTION
**Pull request by eborrallo**
Hello :) , 
I updated the npm packages because if you fetch  the getRecentPerformanceSamples , that is required to calculate TPS, with the actual version fails , and it is fixed on the last versions 
Issue link : https://github.com/solana-labs/solana/pull/26373

And added a new context in the App with a timer interval to call the TPS every time

image

### CODE REVIEW

**Comment by BowTiedHeron** [comment]
Currently the build is failing because of this issue:

Failed to process build result for "api/og-image". Data: {"type":"Lambda"}.
Error: The Serverless Function "api/og-image" is 50.8mb which exceeds the maximum size limit of 50mb. 
Learn More: https://vercel.link/serverless-function-size

@eborrallo @solberenson @gonwi

**Comment by eborrallo** [comment]
Sorry maybe it is a stupid question , but this error is related with the code i added? , because i cant see the relation

**Comment by eborrallo** [comment]
Update PR with the latest changes 
https://github.com/honey-labs/honey-frontend/pull/164

**Comment by eborrallo** [comment]
Update PR with the latest changes

**Comment by eborrallo** [comment]
Fixed , the issue was related with the code on my Refresh context , i just founded other way to do it

File: components/TpsBar/TpsBar.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, Text } from 'degen';
+import { LOW_TPS } from '../../constants/vehoney';
+import { useTPS } from '../../hooks/useTPS';
+
+/**
+ * @params None
+ * @description Range slider regarding borrow and repay feature
```
Comment (BowTiedHeron) [inline components/TpsBar/TpsBar.tsx:8]: I assume you copy pasted the style and forgot to remove this?

Comment (eborrallo) [reply components/TpsBar/TpsBar.tsx]: yes , fixed

**Comment by BowTiedHeron** [comment]
image
Looks dope! @eborrallo 
Would you be able to fit it in the same bar as the wallet connect button and on the left side of the screen?

**Comment by eborrallo** [comment]
> 

Yes i did it  , but what happend on the mobile version? please take a look  when the menu button appear on mobile version

**Comment by BowTiedHeron** [comment]
image
image

I think it looks good! @eborrallo 
What are your thoughts @tomjpandolfi ?

**Comment by BowTiedHeron** [comment]
@eborrallo I just noticed, when I move between pages from farms to governance; the TPS feature bugs out and I have to do a hard refresh. is this something you know how to fix?

**Comment by eborrallo** [comment]
Yep looks like some times on component refresh the function is not called . 
Fixed, now by default i will pick the data from the cache instead of 0

**Comment by BowTiedHeron** [comment]
Closes #150

**Comment by BowTiedHeron** [comment]
Thank you for your contribution to our repo @eborrallo 
Your efforts are much appreciated!!

Now that the PR is merged you should be able to claim the bounty on OpenQ:
https://app.openq.dev/bounty/I_kwDOHB1JPc5L_zuL/0xf48578df83229bd29148af888b675db5e870196f

Don't forget to claim one of our new GitPoaps as well; as proof of your contribution to our repo :)
https://www.gitpoap.io/gp/254

**Pull request merged**